python/pyspark/cloudpickle/compat.py

```diff
@@ -1,13 +1,5 @@
 import sys
 
 
-if sys.version_info < (3, 8):
-    try:
-        import pickle5 as pickle  # noqa: F401
-        from pickle5 import Pickler  # noqa: F401
-    except ImportError:
-        import pickle  # noqa: F401
-        from pickle import _Pickler as Pickler  # noqa: F401
-else:
-    import pickle  # noqa: F401
-    from _pickle import Pickler  # noqa: F401
```

Comment:
From PyPy3.8, `_pickle` is removed.
- https://github.com/cloudpipe/cloudpickle/issues/458